feat: change projects from cards to table

SpyTec opened the pull request:
We were using cards for the list itself, together with no differentiator
for each project, such as their icon, it made it very difficult to
browse in an easy way.

This change moves it over to a composable Table which we can use in
a future change to automatically fetch the branches, releases, and
issues for each project without leaving the page.

However, the table itself offers more information and gives less
confusion as to what each entry is and where to look to see the name --
the repository name will always be on the left side now, which it
didn't before

![image](https://github.com/packit/dashboard/assets/6598829/f00e51ac-68c0-4b54-ad74-018d3b7311e4)

TODO:

- [ ] ‹fill in›

<!-- notes for reviewers -->

<!-- Links to other issues or pull requests,
     for cross-repository links use: ‹namespace›/‹repository›#‹ID of issue›
       (‹namespace›/‹repository›!‹ID of PR› respectively)
-->

Fixes

Related to

Merge before/after

---

RELEASE NOTES BEGIN
Our dashboard now shows projects in a table instead of the cards.
RELEASE NOTES END

mfocko commented:
@SpyTec I reverted an addition of one entry to the gitignore from 519a111. I think it was committed by mistake.

SpyTec commented:
> @SpyTec I reverted an addition of one entry to the gitignore from 519a111. I think it was committed by mistake.

https://github.com/mswjs/msw/discussions/1015

Here is the discussion I found so I would say leave it in. I just thought I added it with a comment before. My bad

mfocko commented:
got it, will add it back
